server/container_status.go

```diff
@@ -18,9 +18,8 @@ func (s *Server) ContainerStatus(ctx context.Context, req *pb.ContainerStatusReq
 		return nil, err
 	}
 
-	if err = s.runtime.UpdateStatus(c); err != nil {
-		return nil, err
-	}
+	// best effort if container is still in runc store...
+	s.runtime.UpdateStatus(c)
```

Comment:
A suggestion here: Could we have `UpdateStatus` return a named error when its `exec.Cmd()` fails, and then ignore that specific error in the 2 cases in this patch ? Otherwise we're ignoring all the other `UpdateStatus()` error paths here.